single-solution/content/gravimetric.php

```diff
@@ -1,14 +1,15 @@
-    <div class="inline-div">
+<div class="inline-div">
 		<input id = "solvent_formula" type="text" placeholder="Solvent Chemical Identity"> <br>
 		<input id = "solute_formula" type="text" placeholder="Solute Chemical Identity"><br>
 		<input id = "solute_molec_weight" type="text" placeholder="Solute Molecular Weight"><br>
         <input id = "solution_concentration" type="text" placeholder="Solution Concentration"><br>
-	</div>
+    <input id= "total_volume" type="text" placeholder="Solution Total Volume (milli Liters)"><br>
+</div>
 
 	<div class="inline-div">
 		<img src="beaker.png" style="width:150px">
         <br>
-		<button id="nextButton" onclick="fill_fields()">Next</button>
+		<button id="nextButton" onclick="next_check('<?php echo $_GET["value"];?>')">Next</button>
 	</div>
```

Comment:
instead of using the built-in nextButton listener function removed in a commit above, that function (including its formula checking code) has been moved into its own "next_check(page)" function, where the page type is passed through params.